examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
It depends on whether we want this to be React-specific, or also work for Angular. I was thinking Angular could then finally use the `.html` file extension. @Urigo, what do you think? If there was a `meteor-platform` without the HTML source handler, would that make the Angular experience better?